cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -988,7 +988,9 @@ func (csr *ClusterStateRegistry) GetClusterSize() (currentSize, targetSize int)
 	for _, accRange := range csr.acceptableRanges {
 		targetSize += accRange.CurrentTarget
 	}
-	currentSize = csr.totalReadiness.Registered - csr.totalReadiness.NotStarted - csr.totalReadiness.LongNotStarted
```

Comment:
Let's update the comment above to clarify this method only returns current and target number of nodes in autoscaled node groups.

While I don't particularly like long names, it also feels like this method's name could better reflect what it's doing - e.g. GetAutoscaledNodesCount?

Reply:
Done.